git_creation_and_tracking/git_creation_and_tracking.md

```diff
@@ -76,13 +82,13 @@ by [b-maze](https://www.deviantart.com/b-maze) / [Deviant Art](https://www.devia
 [Pluto](https://commons.wikimedia.org/wiki/File:PIA19873-Pluto-NewHorizons-FlyingPastImage-20150714-transparent.png) /
```

Comment:
So the b-maze art link is questionably safe for work.  Since we link to the dracula art, and *that* links to the author, can we just say "Brian Maze" without a link?  I didn't check other artists but maybe same policy?

Reply:
@pm0kjp 
Eek! Investigating that image further, it isn't so clear to me that it is CC licensed... although having taken it from SWCarpentry it should be?  I really have no idea how to verify that we can use it at all? Any guidance? 

I've updated the others to just be the image and CC license link.

Reply:
Turns out they don't have a clear record of who/when that image was uploaded, and took it down and created an issue to replace it. So I'm cutting this image out entirely until/unless we make a new one that we can actually use.